src/Matterhorn/Events/Websocket.hs

```diff
@@ -214,7 +214,13 @@ handleWebsocketEvent we = do
             mhLog LogGeneral $ T.pack $
                 "WMTeamDeleted event: " <> show we
 
-        WMUserUpdated -> return ()
+        WMUserUpdated
+            | Just user <- wepUser (weData we) -> do
```

Comment:
Could you factor the body of this case arm out to a function that lives in `Matterhorn.State.Users`? That way this can be abstract w.r.t. the details of the update.

Reply:
I did what I could here, but the `Matterhorn.State.Users` module can't refresh the channel without bigger changes due to cyclic dependencies.  I think just leaving the refresh call with the event is okay, but if you have a better suggestion I'm happy to adjust that.

Reply:
Okay, that makes sense. We already have a handful of `boot` files for resolving import cycles so I don't think it would be much work to fully factor this code out, but I'm okay with it as it is in this patch set. Thanks!